app/views/layouts/angular.html.haml

```diff
@@ -20,11 +20,72 @@
+                  %md_avatar_icon.sidebar__list-item-icon.i.fa.fa-lg.fa-times-circle
+                  %span{translate: "sidebar.muted"}
+            %md_divider.sidebar__divider
+
+            %md_list.sidebar__list.sidebar__groups{ng_class: "{'noGroups': groups().length < 1}"}
+              %md_list_item.sidebar__list-item--group{ng_repeat: "group in groups() | orderBy: 'name' track by group.id", ng_class: "{'selected': onPage('groupPage', group.key)}"}
+                %md_button.sidebar__list-item-button{href: "{{groupUrl(group)}}", "aria-label" => "{{group.name}}"}
+                  %img.md-avatar.lmo-box--tiny.sidebar__list-item-group-logo{ng_src: "{{group.logoUrl()}}"}
+                  %span {{group.name}}
+                %md_list.sidebar__list.sidebar__subgroups{ng_if: "group.subgroups().length > 0"}
+                  %md_list_item.sidebar__list-item{ng_repeat: "subgroup in group.subgroups() | orderBy: 'name' track by subgroup.id"}
+                    %md_button.sidebar__list-item-button--subgroup{href: "{{groupUrl(subgroup)}}"}
+                      %span {{subgroup.name}}
+                .sidebar__list-item-padding
+            %md_divider.sidebar__divider
+            
+            %md_list.sidebar__list.sidebar__explore
+              %md_button.sidebar__list-item-button{href: "/explore", "aria-label" => "{{ 'sidebar.explore' | translate }}"}
+                %md_avatar_icon.sidebar__list-item-icon.i.fa.fa-lg.fa-globe
+                %span{translate: "sidebar.explore"}
+            %md_divider.sidebar__divider
+
+
+            %md_list.sidebar__list.sidebar__user{layout: "column"}
+              %md_list_item.sidebar__list-item--profile{ng_class: "{'selected': onPage('profilePage')}"}
+                %md_button.sidebar__list-item-button{href: "/profile", "aria-label" => "{{ 'sidebar.edit_profile' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.fa.fa-lg.fa-cog
+                  %span{translate: "sidebar.edit_profile"}
+              %md_list_item.sidebar__list-item--email-settings{ng_class: "{'selected': onPage('emailSettingsPage')}"}
+                %md_button.sidebar__list-item-button.sidebar__email-settings{href: "/email_preferences", "aria-label" => "{{ 'sidebar.email_settings' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.fa.fa-lg.fa-envelope-o
+                  %span{translate: "sidebar.email_settings"}
+              %md_list_item.sidebar__list-item--help
+                %md_button.sidebar__list-item-button{href: "{{helpLink()}}", target: "_blank", "aria-label" => "{{ 'sidebar.help' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.fa.fa-lg.fa-info-circle
+                  %span{translate: "sidebar.help"}
+              %md_list_item.sidebar__list-item--contact{ng_if: 'showContactUs()'}
```

Comment:
I think we should stick with `ng-if` / `ng-class` syntax, rather that mixing it up with `_` occasionally.